.az/ios_packaging.yml

```diff
@@ -0,0 +1,91 @@
+# packaging pipeline for iOS CocoaPods package
```

Comment:
let's move this file to .pipeline folder. sooner or later, all pipelines in .az folder will be merged into .pipeline folder which has a better name.